Update Fusion login view setup instructions (#406)
vaadin/docs: articles/fusion/security/spring-login.asciidoc

}
----

Update the `SecurityConfig` to use the `setLogin` helper which will set up everything needed for a Fusion based login view:

> ```java
>   @Override
>   protected void configure(HttpSecurity http) throws Exception {
>     super.configure(http);
>     setLoginView(http, "/login");
>   }
> ```

Reviewer: ```suggestion
[source,java]
----
  @Override
  protected void configure(HttpSecurity http) throws Exception {
    super.configure(http);
    setLoginView(http, "/login");
  }
----
```

== Logout

To handle logging out, you can use the `logout()` helper defined in `auth.ts` above. You typically want to use a button for handling logout and not navigation and a route to avoid timing issues between rendering views and logging out. You can do e.g.

> ```html
> <vaadin-button @click=${() => logout()}>Logout</vaadin-button>
> ```

Reviewer: ```suggestion
[source,html]
----
<vaadin-button @click="${() => logout()}">Logout</vaadin-button>
----
```

== Logout

> To handle logging out, you can use the `logout()` helper defined in `auth.ts` above. You typically want to use a button for handling logout and not navigation and a route to avoid timing issues between rendering views and logging out. You can do e.g.

Reviewer: ```suggestion
To handle logging out, you can use the `logout()` helper defined in `auth.ts` above.
You typically want to use a button for handling logout and not navigation and a route to avoid timing issues between rendering views and logging out.
You can, for example, do the following:
```
